apps/client/src/widgets/community/components/user-comment-list/user-comment-list.tsx

```diff
@@ -43,6 +52,26 @@ const UserCommentList = ({
     }
   }, true);
 
+  const isLiked = feedDetailData?.likedByCurrentUser;
+
+  const { mutate: addLike } = useMutation<LikeAddResponse, Error, void>({
+    ...COMMUNITY_MUTATION_OPTIONS.ADD_LIKE(postId),
+    onSuccess: () => {
+      queryClient.invalidateQueries({
+        queryKey: COMMUNITY_QUERY_KEY.FEED_DETAIL(postId),
+      });
+    },
+  });
```

Comment:
types.ts에서
```tsx
export type LikeAddResponse =
  paths['/posts/{post-id}/likes']['post']['responses']['200']['content']['*/*'];
```
이렇게 data 이전까지만 받아오게 하고
```suggestion
  const { mutate: addLike } = useMutation({
    ...COMMUNITY_MUTATION_OPTIONS.ADD_LIKE(postId),
    onSuccess: () => {
      queryClient.invalidateQueries({
        queryKey: COMMUNITY_QUERY_KEY.FEED_DETAIL(postId),
      });
    },
  });
```
이렇게 사용하시는건 어떨까요? 좋아요 api를 보니 data로 반환되는 값은 `null` 이어서 응답 바디가 {code, message, data: null} 형태라 likeAdd, likeDelete 둘다 `['*/*']`로 두면 전체 바디 타입을 보존할 수 있어요!

Reply:
감사합니다! 뒤에 붙이는 거 싫었는데 저렇게 하니까 깔끔하게 할 수 있네요